jekyll/_cci2/container-runner.adoc

```diff
@@ -95,6 +95,51 @@ agent:
           - name: "other"
 ```
 
+[#unsafe-retries]
+=== Unsafe retries
+
+Unsafe retries enable container runner to automatically rerun tasks if there are unexpected interruptions during their execution. These disruptions could be due to network connectivity issues, the underlying node shutting down, or other unpredictable causes. This feature can also be used when scheduling workloads on spot instances, which often come with cost-saving benefits at the risk of pod preemptions with many Kubernetes providers.
+
+[CAUTION]
+====
+This feature is called “unsafe retries” for a reason. Unlike automatic retries on startup, retrying tasks during runtime can be risky. This is because tasks can have arbitrary steps that produce side effects which are not idempotent or stateless. It is important to use unsafe retries knowing the impact it may have on jobs and workflows. Use this feature with jobs and workflows that are fault-tolerant.
```

Comment:
Instead of noting the impact it will have on jobs and workflows I think we would do well to warn about knowing the impact reran steps may have on external environments and resources such as production environments or databases. 

I'd also change "Use this feature with jobs and workflows that are fault tolerant" to "Use this feature with care, knowing the risks of rerunning jobs and workflows that may or may not be idempotent"

Reply:
This was lifted from Product wording/warning on the feature when previously presented to customers, but I can reword this to that.